insights/parsers/lsblk.py

```diff
@@ -294,17 +294,22 @@ class LSBlockPairs(BlockDevices):
     """
     def parse_content(self, content):
         self.rows = []
+        self.failed_device_paths = set()
```

Comment:
Better add corresponding description for the newly added `failed_device_paths` attribute in above "Attributes:" docstring.